po/es.po

```diff
@@ -121,19 +116,19 @@ msgid "Files:"
 msgstr "Archivos:"
 
 #: newsboat.cpp:122
-#. An actual filename, no translation needed
+#. TRANSLATORS: This is the 'config' filename description used in a table.
```

Comment:
This is going to be removed the next time someone runs `make msgmerge`. Please add these comments to the source code. They should start with three forward slashes for gettext to pick them up.

For these comments to be added to .po files, you'll have to run `make extract && make msgmerge`, but please don't do that: it will create conflicts with current master branch. I'll do this after merging your PR, and then you can start your next PR with an updated version.

Just like before, I've commented about this in a few more places to make it easier to track.